<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionSave_project"/>
   <addaction name="actionSave_project_as"/>
   <addaction name="actionClose_project"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionInformation"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="geometry">
     <rect>
      <x>1711</x>
      <y>108</y>
      <width>176</width>
      <height>222</height>
     </rect>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionSave_project_as"/>
    <addaction name="actionClose_project"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionInformation"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperation"/>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/Resource/icons/new.png</normaloff>:/icons/Resource/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/Resource/icons/open.png</normaloff>:/icons/Resource/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/Resource/icons/save.png</normaloff>:/icons/Resource/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionSave_project_as">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/Resource/icons/save_as.png</normaloff>:/icons/Resource/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save project as ...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/Resource/icons/settings.png</normaloff>
     <normalon>:/icons/Resource/icons/settings.png</normalon>:/icons/Resource/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/Resource/icons/quit.png</normaloff>:/icons/Resource/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionClose_project">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/Resource/icons/close.png</normaloff>:/icons/Resource/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close project</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/Resource/icons/start.png</normaloff>:/icons/Resource/icons/start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/Resource/icons/stop.png</normaloff>:/icons/Resource/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="whatsThis">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/Resource/icons/information.png</normaloff>:/icons/Resource/icons/information.png</iconset>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
